components/FButton.vue

```diff
@@ -451,6 +452,16 @@ const emit = defineEmits<{
   (name: 'blur'): void;
 }>();
 
+const buttonRef = ref<HTMLButtonElement>();
+
+defineExpose<{
+  ref: Ref<HTMLButtonElement | undefined>;
+  focus: () => void;
+}>({
+  ref: buttonRef,
+  focus,
+});
+
```

Comment:
- where is defined `focus` ?
- if it was defined, why expose `ref` instead to handle the focus of the button outside ?
- if you still need `ref`, keep it `buttonRef` for clarity